### PR TITLE
Use force overwrite flag in create_symlinks

### PR DESCRIPTION
**Pull request by jandersson**
I can't identify a way to setup dependency without it downloading all the scripts. So the -f flag is needed so the existing scripts get overwritten, else you get this error: 
```
ln: failed to create symbolic link '../scripts/pick.lic': File exists
ln: failed to create symbolic link '../scripts/pilgrimage.lic': File exist
```

Even running dependency without the install command will download the scripts (where here it is replacing scripts i had symlinked beforehand). 
```
;dependency
--- Lich: dependency active.
[dependency: Installing rest-firebase gem, this may take a few minutes.]
>
[dependency: Finished Installing rest-firebase gem.]
>
[dependency: Renaming existing script accept-sell.lic to accept-sell.lic1476532733.bak]
[dependency: Renaming existing script addroom.lic to addroom.lic1476532734.bak]
>
[dependency: Renaming existing script afk.lic to afk.lic1476532735.bak]
[dependency: Renaming existing script appraisal.lic to appraisal.lic1476532736.bak]
[dependency: Renaming existing script astrology.lic to astrology.lic1476532736.bak]
>
[dependency: Renaming existing script attunement.lic to attunement.lic1476532737.bak]
[dependency: Renaming existing script automap.lic to automap.lic1476532738.bak]
>
[dependency: Renaming existing script bescort.lic to bescort.lic1476532739.bak]
[dependency: Renaming existing script better-book.lic to better-book.lic1476532740.bak]
[dependency: Renaming existing script bootstrap.lic to bootstrap.lic1476532741.bak]
[dependency: Renaming existing script buff.lic to buff.lic1476532741.bak]
 * Hoshino joins the adventure.
>
```

### CODE REVIEW

File: create_symlinks.sh [hunk 1]
```diff
@@ -9,5 +9,5 @@ destination=$1
 find $1 -maxdepth 1 -type l -exec rm -f {} \;
 
 # Link all the things
-ln -s `pwd`/*.lic $1
+ln -sf `pwd`/*.lic $1
 ln -s `pwd`/profiles/ $1
```
Comment (rpherbig) [inline create_symlinks.sh]: Do you also need the `f` flag for /profiles/ ?

Comment (jandersson) [inline create_symlinks.sh]: I think the user would lose their existing profiles then

Comment (jandersson) [inline create_symlinks.sh]: I symlink my profiles (from a separate directory) into dr-scripts/profiles after running this script. Not sure if thats the best way to do handle it, but it seems to work

**Comment by rpherbig** [comment]
@kamarcum you wrote this script, any thoughts on this change?

**Comment by jandersson** [comment]
Going to close this so I can fix my master branch (all those merge messages are bothering me). I'll re-open another later.

**Pull request closed without merging**